samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -52,6 +52,10 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       adapter.onStateChange((state: CallAdapterState) => {
         const pageTitle = convertPageStateToString(state);
         document.title = `${pageTitle} - ${webAppTitle}`;
+
+        if (state.call?.state === 'Connected') {
```

Comment:
Is callId different to groupId/teamsMeetingLink? (the information in url query param?)

Reply:
callId is the internal call, it doesnt have anything to do with the groupid or teamsMeeting link